src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -103,8 +113,30 @@ protected internal override PinMode GetPinMode(int pinNumber)
 
         protected internal override bool IsPinModeSupported(int pinNumber, PinMode mode)
         {
-            // Libgpiod Api do not support pull up or pull down resistors for now.
-            return mode != PinMode.InputPullDown && mode != PinMode.InputPullUp;
+            switch (mode)
+            {
+                case PinMode.Input:
+                    return true;
+                case PinMode.InputPullDown:
+                case PinMode.InputPullUp:
+                    // for use the bias flags we need Kernel version 5.5 or later
```

Comment:
Correct me if I'm wrong but I believe we would also need libgpiod version to be 1.5 correct? That one will be a bit harder to check, but we could try calling the new PInvoke inside a try catch to see if the method is present on the current installation of libgpiod.

Reply:
Yes, correct. What do you think, keep the check for the kernel, removing the Regex, or just the check for bind libgpiod?

Reply:
Well, the idea of this method is to return whether or not a pin mode is supported on the current hardware, so in order for us to tell the truth here, we would have to make both checks, that is to ensure Kernel is 5.5 plus also ensure libgpiod version is recent enough.

Reply:
The libpgiod has a function that return the version, so I thought it better to use it in checks. I add the binding on 47ff809

Reply:
Thanks for review @joperezr . I also added the check for the libgpiod in IsPinModeSupported d28e719. Let me know what you think.